src/types.ts

```diff
@@ -158,13 +158,15 @@ export type ExcalidrawAPIRefValue =
       ready?: false;
     };
 
-
 export type SaveBoardEventArgs = {
-  elements: any,
-  snapshot: any,
-  state: any
+  elements: any;
+  snapshot: any;
+  state: any;
 };
 
+/** The frequency to fire the onSaveNeededEvent (if applicable) */
+export type SaveFrequency = "ALL" | "ONEXIT" | "NONE";
+
```

Comment:
Here we can also do:
```suggestion
/** The frequency to fire the onSaveNeededEvent (if applicable) */
export enum SaveFrequency {
  All= 'ALL',
  OnExit='ONEXIT',
  None= 'NONE'
}
```
And use as
```
        ladumaApi.getSaveFrequency() === SaveFrequency.All
```
which has the advantage of leveraging TS' typeahead. Also, if in the future we have to change one of the enum's elements for some reason, will save us from changing in multiple locations.

Reply:
thanks @carboni-rob , I thought about using enum (and in fact I did start using it this way), but then I went in the other direction for no specific reason! :) . Are you happy if I raise this and the name correction above as a small separate code tidy up task?